docs/source/zero-code-change-logging.rst

```diff
@@ -0,0 +1,111 @@
+Understanding Acceleration and Fallbacks with Logging
+====================================================
+
+``cuml.accel`` provides comprehensive logging to help you understand which
+operations are being accelerated on GPU and which are falling back to CPU
+execution. This can be particularly useful for debugging performance issues
+or understanding why certain operations might not be accelerated.
+
+To enable logging, you can set the logging level in several ways:
+
+**Command Line Interface (CLI):**
```

Comment:
Can we use (sub)headings instead of boldness? That way we can link to things from other parts of the docs or when pointing someone to the docs with a URL